centos.org/ansible/foreman.duffy.yml

```diff
@@ -0,0 +1,5 @@
+plugin: evgeni.duffy.inventory
```

Comment:
right now this inventory would return *all* our nodes, so parallel jobs would be "funny", I need to add a limit to the ansible run

Reply:
Is this what the PR is waiting on to move out of draft?

Reply:
correct.

Reply:
this is now "resolved" as I added code that should do what I want, but didn't test it yet